cmd/root.go

```diff
@@ -0,0 +1,67 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var cfgFile string
+
+var RootCmd = &cobra.Command{
+	Use:   "wal-g",
+	Short: "PostgreSQL backup tool",
+	Long: `A longer description that spans multiple lines and likely contains
```

Comment:
We need at least some text here

Reply:
This PR is devoted to refactoring. Let's make an issue about the appropriate description to solve it in another PR?

Reply:
OK